test/Hydration-test.js

```diff
@@ -0,0 +1,111 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Hydration = require('../src/Hydration.js');
+
+describe('Hydration', function() {
+  let hydrationData;
+
+  beforeEach(function() {
+    hydrationData = [
+      {
+        "userID": 1,
+        "date": "2019/06/15",
+        "numOunces": 37
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/15",
+        "numOunces": 75
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/16",
+        "numOunces": 69
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/16",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/17",
+        "numOunces": 96
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/18",
+        "numOunces": 100
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/19",
+        "numOunces": 20
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/20",
+        "numOunces": 31
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/21",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/22",
+        "numOunces": 55
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/23",
+        "numOunces": 74
+      }
+    ];
+
+    userOneHydration = new Hydration(hydrationData, 1);
+    userTwoHydration = new Hydration(hydrationData, 2);
+  });
+
+  it.skip('should be a function', function() {
+    expect(Hydration).to.be.a('function');
+  });
+
+  it.skip('should instantiate Hydration', function() {
+    expect(userOneHydration).to.be.an.instanceOf(Hydration);
+  });
+
+  it.skip('should only have hydration stats for one user', function() {
+    expect(userOneHydration.hydrationCalendar.length).to.equal(3);
+    expect(userTwoHydration.hydrationCalendar.length).to.equal(3);
+  });
+
+  it.skip('should have users hydration stats for all recorded dates', function() {
+    expect(userOneHydration.hydrationCalendar).to.have.all.keys('2019/06/15', '2019/06/16', '2019/06/17');
+  });
```

Comment:
We want keys to be dates -- but maybe our dates should be in a different format? Something easier to sort?

Reply:
I can do some research later on this. I assume their is some sort of convention for using dates in JS.